src/realm/object-store/shared_realm.hpp

```diff
@@ -341,16 +341,32 @@ class Realm : public std::enable_shared_from_this<Realm> {
     // WARNING / FIXME: compact() should NOT be exposed publicly on Windows
     // because it's not crash safe! It may corrupt your database if something fails
     bool compact();
-    // For synchronized realms, the file written will have the client file ident removed. Furthermore
-    // it is required that all local changes are synchronized with the server before the copy can
-    // be written. This is to be sure that the file can be used as a stating point for a newly
-    // installed application. The function will throw if there are pending uploads.
+    // Deprecated. Use Realm::convert instead.    
     void write_copy(StringData path, BinaryData encryption_key);
```

Comment:
If we're not deprecating these, they should be made internal/private.

Reply:
Since the new function is called differently anyway, we might just remove them alltogether.